### ISSUE TITLE
Error when a user without access to notes tries to move a person to a new family

### ISSUE THREAD

**Issue by tbar0970** [tbar0970/jethro-pmm#115]
```
Current user has insufficient permission level to load a Family_Note object 
Line 26 of /home/jethro/code/2.13.2/include/db_object.class.php

Request: Array
(
   [view] => _move_person_to_family
   [personid] => 1001
   [move_to] => existing
   [familyid] => 541
)


Array
(
   [0] => Array
       (
           [function] => _handleError
           [class] => System_Controller
           [type] => ->
           [args] => Array
               (
                   [0] => 256
                   [1] => Current user has insufficient permission level to load a Family_Note object
                   [2] => /home/jethro/code/2.13.2/include/db_object.class.php
                   [3] => 26
                   [4] => [Object/Array]
               )

       )

   [1] => Array
       (
           [file] => /home/jethro/code/2.13.2/include/db_object.class.php
           [line] => 26
           [function] => trigger_error
           [args] => Array
               (
                   [0] => Current user has insufficient permission level to load a Family_Note object
                   [1] => 256
               )

       )

   [2] => Array
       (
           [file] => /home/jethro/code/2.13.2/views/view_0_move_person_to_family.class.php
           [line] => 41
           [function] => __construct
           [class] => db_object
           [type] => ->
           [args] => Array
               (
               )

       )

   [3] => Array
       (
           [file] => /home/jethro/code/2.13.2/include/system_controller.class.php
           [line] => 120
           [function] => processView
           [class] => View__Move_Person_To_Family
           [type] => ->
           [args] => Array
               (
               )

       )

   [4] => Array
       (
           [file] => /home/jethro/code/2.13.2/index.php
           [line] => 58
           [function] => run
           [class] => System_Controller
           [type] => ->
           [args] => Array
               (
               )

       )

)
```

**Comment by tbar0970** [comment]
How to resolve?
a) don't try to move the notes if current user can't see them.  Probably a bad idea.
b) only allow users who can see notes move persons between families
c) somehow suspend the security rules during the operation so that the notes can be moved.

**Comment by tbar0970** [comment]
Was fixed in 10e779e
It wasn't to do with notes being moved, but rather a new note being added
